packages/timeline-state-resolver/src/service/device.ts

```diff
@@ -74,7 +74,7 @@ export interface DeviceEvents {
 
 	debugState: [state: object]
 	/** The connection status has changed */
-	connectionChanged: [status: DeviceStatus]
+	connectionChanged: [status: Omit<DeviceStatus, 'active'>]
```

Comment:
At this point we should probably have `DeviceStatus` extend another interface

Reply:
I did think that, but didn't want to figure that out as part of this